src/ceph-disk/ceph_disk/main.py

```diff
@@ -5389,6 +5399,12 @@ def make_activate_space_parser(name, subparsers):
         metavar='DEV',
         help='path to %s block device' % name,
     )
+    activate_parser.add_argument(
```

Comment:
s/activate_space_parser/activate_parser/

Reply:
fixed the name